volumina/tiling/__init__.py

```diff
@@ -315,6 +317,12 @@ def requestRefresh(self, rectF, stack_id=None, prefetch=False, layer_indexes=Non
         """
         stack_id = stack_id or self._current_stack_id
         tile_nos = self.tiling.intersected(rectF)
+        to_cancel = self._current_requests
+        self._current_requests = {}
```

Comment:
Volumina's code is kinda hard to navigate, so I'd love to see some type annotations like

`self._current_requests: Mapping[Tuple[LAYER_ID, ImageSource, REQUEST_ID], ArrayRequest] = {}`